genvm/templates/multisig/gas.go

```diff
@@ -0,0 +1,35 @@
+package multisig
+
+import (
+	"math"
+
+	"github.com/spacemeshos/go-spacemesh/genvm/core"
+)
+
+func BaseGas(method uint8, signatures int) uint64 {
+	switch method {
+	case core.MethodSpawn:
+		return core.TX + core.EDVERIFY*uint64(signatures) + core.SPAWN
+	case core.MethodSpend:
+		return core.TX + core.EDVERIFY*uint64(signatures)
+	}
+	return math.MaxUint64
+}
+
+func LoadGas(keys int) uint64 {
+	return core.ACCOUNT_ACCESS + core.SizeGas(core.LOAD, 32*keys+16)
```

Comment:
would appreciate if you can comment on what 16 is (and other constants in this PR), like the comments you have in vesting.go.

Reply:
right, wanted to add and forgot